Realm/RLMSyncUser.h

```diff
@@ -35,6 +35,15 @@ typedef NS_ENUM(NSUInteger, RLMSyncUserState) {
 /// A block type used for APIs which asynchronously vend an `RLMSyncUser`.
 typedef void(^RLMUserCompletionBlock)(RLMSyncUser * _Nullable, NSError * _Nullable);
 
+/// A block type used to report the progress of a permissions set operation.
```

Comment:
Actually, "progress" is the wrong word here as the operation has always _completed_ when the callback is invoked, it just may or may not have succeeded. So instead:

"A block type used to report the status of a permissions set operation."